Resources/config/services.xml

```diff
@@ -5,23 +5,16 @@
     <services>
         <!-- sulu-admin -->
         <service id="sulu_redirect.admin" class="Sulu\Bundle\RedirectBundle\Admin\RedirectAdmin" public="false">
+            <argument type="service" id="sulu_admin.route_factory"/>
```

Comment:
@alexander-schranz Still not sure if this is a good idea. If the route builder factory is overridden by just changing the alias (which I would consider a valid thing to do) this class would still use the core service, instead of the overridden one.

Are there any real reasons for not doing this, besides the pure fact that we should not use FQCN?

Reply:
We should definitly following best practices and if somebody want to overwrite it for all bundles it should overwrite it by using the service id and not the FQCN.

Reply:
Where is written that this is best practice? I only know about [the part of the symfony documentation saying that bundles should not rely on autowiring](https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles) (which we don't do here), and it says nothing about how services should be named.

Reply:
And thats Service named should ne prefixed e.g. acme_blog. 

If the bundle defines services, they must be prefixed with the bundle alias. For example, AcmeBlogBundle services must be prefixed with acme_blog.